Src/VimCore/FSharpUtil.fs

```diff
@@ -584,7 +584,7 @@ module CharUtil =
     let IsTagNameChar x = System.Char.IsLetterOrDigit(x) || x = ':' || x = '.' || x = '_' || x = '-'
     let IsFileNameChar x = IsTagNameChar x
     let IsPrintable x =
-        let category = CharUnicodeInfo.GetUnicodeCategory(x)
+        let category = CharUnicodeInfo.GetUnicodeCategory(x: char)

```

Comment:
Were the type's necessary here and forced by an F# change in inference?

Reply:
although now I just looked at that code again, it would probably be neater to add the type annotation to `IsPrintable x` instead.

Reply:
Either way is fine. I'm just always curious if it's a bug / broken behavior the code was relying on in F# or if there was a diff reason. Found a few times where I had relied on F# bugs.